### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by Ivanopolus21**

### CODE REVIEW

File: Tasks/skip.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+
 // Remove values from array
 
-Skip =(T,...Remove)=> {
-x = 0;
-for (C of T) {
-for (X of Remove) {
-if (C === X) {
-  T.splice(x, 1);
+const skip = (array, ...charsToRemove) => {
```
Comment (tshemsedinov) [inline Tasks/skip.js:5]: Why chars?

File: Tasks/words.js [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-   if (!flag) {
-   if (c === ' ') {
-   if (flag === true) { flag = false } else { flag = false }
-    } else {
-   if (flag === true) { flag = true } else { flag = true }
-      number_of_words_in_s++
+const words = (string) => {
+  let numberOfWords = 0;
+  let flag = false;
+  for (const element of string) {
```
Comment (tshemsedinov) [inline Tasks/words.js:8]: Here `char` will be ok )

File: Tasks/skip.js [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
+  let removedChar = 0;
+  for (const selectedChars of array) {
+    for (const selectedCharsToRemove of charsToRemove) {
```
Comment (tshemsedinov) [inline Tasks/skip.js:8]: Try to avoid nested loops

File: Tasks/skip.js (hunk 1 above)
Comment (Ivanopolus21) [reply Tasks/skip.js]: Was changed to values!

File: Tasks/words.js (hunk 2 above)
Comment (Ivanopolus21) [reply Tasks/words.js]: Good idea, thank You! Was changed!

File: Tasks/skip.js (hunk 3 above)
Comment (Ivanopolus21) [reply Tasks/skip.js]: Got it! Tried to remove one loop and use some methods instead

**Pull request closed without merging**